src/index.html

```diff
@@ -7,7 +7,326 @@
+            <p class="card__job">
+              CEO
+            </p>
+            <p class="card__txt">
+              Nice work, Certificates National Association of Realtors (USA)
+            </p>
+          </div>
+          <div class="card testimonials__card">
+            <img src="./images/people/2.jpg" alt="Scarlett Hawkins"
+              class="card__avatar">
+            <h3 class="card__name">
+              Scarlett Hawkins
+            </h3>
+            <p class="card__job">
+              Senior Vice President
+            </p>
+            <p class="card__txt">
+              My soul is illuminated by an unearthly joy, as these beautiful
+              spring morning, which I enjoy with all my heart
+            </p>
+          </div>
+
+          <div class="card testimonials__card">
+            <img src="./images/people/3.jpg" alt="Max Henry"
+              class="card__avatar">
+            <h3 class="card__name">
+              Max Henry
+            </h3>
+            <p class="card__job">
+              Chief Design Officer
+            </p>
+            <p class="card__txt">
+              When my lovely valley of steam rises and half-day sun is above
+              an
+              impermeable
+            </p>
+          </div>
+        </div>
+      </section>
+
+      <section class="contacts page-content__container">
+        <h2 class="section-title contacts__section-title">
+          Contact us
+        </h2>
+
+        <div class="contacts__inner">
+          <div class="contacts__wrapper">
+            <div class="contacts__phone-email">
+              <h3 class="contacts__subtitle">
+                Phone
+              </h3>
+              <a class="contacts__link" href="#">
+                +1 234 5555-55-55
+              </a>
+
+              <h3 class="contacts__subtitle">
+                Email
+              </h3>
+              <a class="contacts__link" href="#">
+                hello@miami.com
+              </a>
+            </div>
+            <div class="contacts__address">
+              <h3 class="contacts__subtitle">
+                Address
+              </h3>
+              <a class="contacts__link" href="#">
+                400 first ave. n. suite 700 Minneapolis, MN 55401
+              </a>
+            </div>
+          </div>
+
+          <form action="" class="contacts__form">
+            <input
+              name="Name"
+              type="text"
+              class="input contacts__input"
+              placeholder="Name"
+              required
+            >
+            <input
+              name="Email"
+              type="email"
+              class="input contacts__input"
+              placeholder="Email"
+              required
+            >
+            <textarea
+              name="Message"
+              type="text"
+              rows="8"
+              class="input contacts__input"
+              placeholder="Message"
+            >
+            </textarea>
```

Comment:
```suggestion
           ></textarea>
```
так будет отображаться placeholder и не будет ругаться линтер